tftrt/examples/image-classification/image_classification.py

```diff
@@ -51,9 +54,13 @@ def after_run(self, run_context, run_values):
             print("    step %d/%d, iter_time(ms)=%.4f, images/sec=%d" % (
                 current_step, self.num_steps, duration * 1000,
                 self.batch_size / self.iter_times[-1]))
+        self.iteration_number += 1
```

Comment:
Isn't it better to make a new Hook for `iteration_number`?

We made `LoggerHook` to improve logging, but `iteration_number` is actually used for stopping the benchmark.